src/client/cmd/start.cpp

```diff
@@ -59,25 +61,19 @@ mp::ReturnCode cmd::Start::run(mp::ArgParser* parser)
                                "instance.\n");
                 }
             }
-            else
-            {
-                mp::MountError mount_error;
-                mount_error.ParseFromString(status.error_details());
-
-                if (mount_error.error_code() == mp::MountError::SSHFS_MISSING)
-                {
-                    cmd::install_sshfs_for(mount_error.instance_name(), parser->verbosityLevel(), rpc_channel, stub,
-                                           cout, cerr);
-                }
-            }
         }
 
         return ret;
     };
 
+    auto streaming_callback = [&spinner](mp::StartReply& reply) {
+        spinner.stop();
```

Comment:
I wonder if `start` should just restart in the animated spinner, rather than requiring stop/start calls.

Reply:
Yeah, probably some sort of check that is the message string is different, then it resets it or something...but for a different PR:)